### DISCUSSION TITLE
PMM seems to have run with no error.  Found content, applied overlays.  No overlays show in plex.

### DISCUSSION THREAD

**Discussion by adam-rudd** [meisnate12/Plex-Meta-Manager#1986] [Q&A] [answered]
Setup wasn't too bad, used the docker-compose method.  Some small hicups but no big deal, built out a config file, dropped in plex and tmdb apikey. Had a little bit of confusion until I found the two different ssl flags, set both to false.  Connected to plex right away.  

Seems to have run with no errors (lots of ssl errors, but should be ignored).  But no overlays show in plex.  

I refreshed page.  Tried on a different machine.  Stopped and restarted plex server.  Nothing:  Test library:  

Is there another step I'm missing? Or env variable?  

[meta1.log](https://github.com/meisnate12/Plex-Meta-Manager/files/15045148/meta1.log)

**Comment by chazlarson** [comment]
PMM goes through and figures out which overlays go on which items, then when it tries to apply overlays, it gets an ssl error on every single attempt to retrieve the base artwork from Plex:
```
| requests.exceptions.SSLError: HTTPSConnectionPool(redacted): Max retries exceeded with url: /library/metadata/30913/thumb/1713548177?X-Plex-Token=(redacted) (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: IP address mismatch, certificate is not valid for '192.168.10.20'. (_ssl.c:1002)"))) |
```

**Comment by adam-rudd** [comment, marked as answer]
Got it.  Continued my search.  Found this.  Used an instance of Tautulli https://github.com/meisnate12/Plex-Meta-Manager/issues/1838#issuecomment-1930931192
